Packs/Core/Playbooks/playbook-Exchange_forwarding_rule_configured.yml

```diff
@@ -0,0 +1,1797 @@
+    type: regular
+    task:
+      id: 3fa5a92e-3b86-4a05-8b86-53cd466bb1cb
+      version: -1
+      name: Get caller IP and forwarding mail address
+      description: Returns information about each alert ID.
+      script: '|||core-get-cloud-original-alerts'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "2"
+      - "28"
+      - "6"
+    scriptarguments:
+      alert_ids:
+        simple: ${alert.id}
+      filter_alert_fields:
+        simple: "false"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": -180
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: be117be2-af06-4d9e-8b01-19cc4b115d02
+    type: regular
+    task:
+      id: be117be2-af06-4d9e-8b01-19cc4b115d02
+      version: -1
+      name: 'Check caller IP reputation '
+      description: Checks the reputation of an IP address.
+      script: '|||ip'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "4"
+    scriptarguments:
+      ip:
+        complex:
+          root: Core.OriginalAlert.event
+          accessor: caller_ip
+          transformers:
+          - operator: uniq
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -160,
+          "y": -10
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: b504ebe2-56d0-400e-8222-a7d57b546615
+    type: regular
+    task:
+      id: b504ebe2-56d0-400e-8222-a7d57b546615
+      version: -1
+      name: Check forwarding email Domain reputation
+      description: Checks the reputation of a domain.
+      script: '|||domain'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "4"
+    scriptarguments:
+      domain:
+        complex:
+          root: Core.OriginalAlert.raw_abioc.event
+          accessor: forwarding_domain_with_tld
+          transformers:
+          - operator: uniq
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 240,
+          "y": 180
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: 0b23837d-9e34-4468-8c26-2d54a9705b83
+    type: condition
+    task:
+      id: 0b23837d-9e34-4468-8c26-2d54a9705b83
+      version: -1
+      name: Evaluate domain and IP address risk level
```

Comment:
How this conditions handles a malicious domain if we only block the IP afterwards? I don't see that BlockIP subplaybooks gets the malicious domain in case

Reply:
We are not blocking any domain because this is an email domain. But I am checking if it is malicious, and if so, I block the IP. We discussed this in Arik's demo.